tests/modules/images/fetchpriority/fetchpriority-tests.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Tests the adding of fetchpriority to img tags in the content.
+ *
+ * @Covers ::wp_filter_content_tags
```

Comment:
Test is for `fetchpriority_img_tag_add_attr` function isn't it? And should use lowercase for `@covers` annotation.

```suggestion
 * @covers ::fetchpriority_img_tag_add_attr
```